addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -692,7 +692,8 @@ def reset_applicant(self):
         for applicant in self:
             applicant.write(
                 {'stage_id': applicant.job_id.id and default_stage[applicant.job_id.id],
-                 'refuse_reason_id': False})
+                 'refuse_reason_id': False,
+                 'referral_state': 'progress'}})
```

Comment:
why do you target 18.0 ? The commit you referenced is in the code base since odoo 14.0. :thinking: